File: app/src/main/java/com/sdapps/formula1fy/core/Commons.java
package com.sdapps.formula1fy.core;

import android.content.Context;
import android.util.Log;


public class Commons {

    private Context context;

    public Commons(Context context){
        this.context = context;

    }

    public void print(String data){
        Log.d(this.getClass().getName(),data);
    }
}

File: app/src/main/java/com/sdapps/formula1fy/core/utils/Commons.kt
package com.sdapps.formula1fy.core.utils

import android.util.Log

class Commons {
    fun print(data: String?) {
        Log.d(this.javaClass.name, data!!)
    }

    fun printException(data: Throwable?){
        Log.e(this.javaClass.name, data.toString())
    }
}
